src/components/_pagination.scss

```diff
@@ -6,6 +6,111 @@
  * of all or any part of the contents of this file is strictly prohibited.
  */
 
+nav.pagination {
+  position: relative;
+  height: $input-height;
+  justify-content: flex-end;
+
+  > span.fade.show {
+    white-space: nowrap;
+    align-self: center;
+    margin-right: px(4);
+  }
+
+  > div.fade {
+    position: absolute;
```

Comment:
The transition looks good. However I don't like the magic calculation (`right: $input-height * 2;`).
I would prefer setting the `display` prop or playing a bit with animations.

Reply:
That calculation has nothing to do with the animation. It just shifts the absolutely positioned "Show: [] Go: []" controls to the left past the Prev/Next arrows. Each of those arrows is $input-height wide, so multiplying by 2 gives us the correct amount to shift over. So it's not magic.

Reply:
- The fade in/out is already an effect. Not sure what other effect we would want, but we could achieve most simple effects via the use of transitions. I don't see the need for keyframe animations.
- I'm not really understanding your other points here. Maybe we can discuss when you have the time.